Ladybird/Qt/WebContentView.cpp

```diff
@@ -395,13 +368,40 @@ void WebContentView::mouseReleaseEvent(QMouseEvent* event)
     }
     auto modifiers = get_modifiers_from_qt_mouse_event(*event);
     auto buttons = get_buttons_from_qt_event(*event);
-    client().async_mouse_up(to_content_position(position), screen_position, button, buttons, modifiers);
+    client().async_mouse_up(position, screen_position, button, buttons, modifiers);
+}
+
+void WebContentView::wheelEvent(QWheelEvent* event)
```

Comment:
is there any reason to move this method around?

Reply:
Oh yes I moved the them to the same order of `WebContentServer.ipc`. If you don't like it I can place them back

Reply:
Generally it's better to not move and modify code in the same commit. It makes it harder to see what the changes are to review them.

Reply:
You are right, I wouldn't do it again 😉